src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -332,30 +332,49 @@ namespace {
 
 void TwitchIrcServer::bulkRefreshLiveStatus()
 {
-    QStringList userIDs;
-    this->forEachChannel([&userIDs](ChannelPtr chan) {
+    auto twitchChans = std::make_shared<QHash<QString, TwitchChannel *>>();
```

Comment:
When I was writing code for this PR I've noticed that without making QHash a pointer, it was copied in onSuccess and finally callbacks, making `twitchChans->remove(...);` call do nothing since it operated on a copied value.